src/main/java/rueppellii/backend2/tribes/kingdom/Kingdom.java

```diff
@@ -7,6 +7,7 @@
 
 import javax.persistence.*;
```

Comment:
Try to not import all the elements of the library, just the necessary ones

Reply:
Jupiter is the latest version thats why I removed everything related to the previous versions and it was working perfectly.